scapy/arch/common.py

```diff
@@ -15,6 +15,11 @@
 from scapy.config import conf
 import scapy.modules.six as six
 
+try:
+    long        # Python 2
+except NameError:
+    long = int  # Python 3
```

Comment:
long is used in a compatibility function only used with Python 2. This should not be needed